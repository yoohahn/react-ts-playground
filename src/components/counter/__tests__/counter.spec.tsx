import React from "react";
import ReactDOM from "react-dom";
import TestUtils from "react-dom/test-utils";
import Counter from "../index";

jest.useFakeTimers();

describe("<Counter />", () => {
  it("Counter to render with start time 10", () => {
    const counter = TestUtils.renderIntoDocument(<Counter start={10} /> as any);
    const counterNode = ReactDOM.findDOMNode(counter);
    expect(counterNode).toMatchSnapshot();
  });
});
