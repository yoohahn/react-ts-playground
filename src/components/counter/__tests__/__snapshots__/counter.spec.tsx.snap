// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`<Counter /> Counter to render with start time 10 1`] = `
<div
  class="counter"
>
  10
</div>
`;
